### DISCUSSION TITLE
Any way to locally build for other OSes without using Docker?

### DISCUSSION THREAD

**Discussion by nikspyratos** [php/frankenphp#371] [Q&A] [answered]
Just curious if this is a possibility. Had a look through the `build_static.sh` file but I don't understand it very well just yet. If I understand correctly no actual OS/arch arguments are passed into the build commands, so by default it spits out a binary for whatever system you're on.

Would be neat for those who don't want local Docker VM overheads on mac to be able to do this.

**Comment by dunglas** [comment, marked as answer]
Cross-compiling is in theory possible, but AFAIK not (yet) supported by `static-php-cli` (I tried, but didn't manage to make it working).

**Reply by nikspyratos** [reply]
Ah thanks for the info!
